pkg/option/config.go

```diff
@@ -1493,6 +1496,11 @@ type DaemonConfig struct {
 	// endpoint may allow traffic to exchange traffic with.
 	PolicyMapEntries int
 
+	// PolicyStatsMapEntries is the maximum number of stats entries kept
+	// globally in the node in an LRU map, where oldest entries are recycled
+	// to make space for new ones.
+	PolicyStatsMapEntries int
```

Comment:
Let's also avoid adding new stuff to `option.Config` if we can avoid it, please. If you make the policy stat map a cell, I think you can declare your config there and won't have to add a new field here.

I haven't checked whether you need access to this field somewhere which doesn't have access to the hive - if so, that may be trickier.

Reply:
Moved to the new Cell.